src/main/webapp/resources/js/components/samples/components/SampleFileList.tsx

```diff
@@ -17,20 +17,31 @@ export function SampleFileList() {
   const dispatch = useDispatch();
   const [removeSampleFilesFromSample] = useRemoveSampleFilesMutation();
 
-  const { sample } = useSelector((state) => state.sampleReducer);
-  const { files } = useSelector((state) => state.sampleFilesReducer);
+  const { sample } = useSelector(
+    (state: RootStateOrAny) => state.sampleReducer
+  );
+  const { files } = useSelector(
+    (state: RootStateOrAny) => state.sampleFilesReducer
+  );
 
   /*
   Remove sequencing objects and/or genome assembly objects from sample
    */
-  const removeSampleFiles = ({ fileObjectId, type }) => {
+  const removeSampleFiles = ({
+    fileObjectId,
+    type,
+  }: {
+    fileObjectId: string | number;
```

Comment:
Why can these be either?

Reply:
It's supposed to be just one of them. Looks like one of those things I let intellij add for me

Reply:
Updated in 445c58cec70fbfbbe6113426d30eb0caf860e90f